src/dagre-js/create-edge-labels.js

```diff
@@ -3,9 +3,9 @@ import _ from 'lodash-es';
 import { addLabel } from "./label/add-label";
 import * as util from "./util";
 
-export { createEdgeLabels };
+export { createEdgeLabels, setCreateEdgeLabels };
 
-function createEdgeLabels(selection, g) {
+var createEdgeLabels = function(selection, g) {
```

Comment:
Do you mind using `const` instead of `var`?

Reply:
https://github.com/tbo47/dagre-es/pull/3#discussion_r1014727984